ipatests/test_ipapython/test_certdb.py

```diff
@@ -242,3 +244,14 @@ def test_delete_cert_and_key():
             assert len(nssdb.list_certs()) == 1
     finally:
         os.unlink(p12file)
+
+
+def test_check_validity():
```

Comment:
@tiran thanks for the PR.  My only comment is that this test (thanks for adding it) does not exercise the new NotBefore/NotAfter check.  Ideally we test that here, or in an integration test, or both.